### ISSUE TITLE
axios instance (using alias) typing: `headers` is required althought all the headers are optional

### ISSUE THREAD

**Issue by vltansky** [ecyrbe/zodios#298]
```
Argument of type '{ params: { rReferenceId: number; }; queries: { queryparams: { loanAmountTo: number; } | {}; }; }' is not assignable to parameter of type 'DeepReadonlyObject<{ headers: { "X-Session-ID"?: string; "X-Aid"?: string; "X-Channel-ID"?: string; "X-Message-ID"?: string; "X-Transaction-ID"?: string; "X-Uid"?: string;  } & Partial<...> & AxiosHeaders; ... 32 more ...; formSerializer?: FormSeria...'.
  Property 'headers' is missing in type '{ params: { rReferenceId: number; }; queries: { queryparams: { loanAmountTo: number; } | {}; }; }' but required in type 'DeepReadonlyObject<{ headers: { "X-Session-ID"?: string; "X-Aid"?: string; "X-Channel-ID"?: string; "X-Message-ID"?: string; "X-Transaction-ID"?: string; "X-Uid"?: string;  } & Partial<...> & AxiosHeaders; ... 32 more ...; formSerializer?: FormSeria...'.ts(2345)
zodios.types.d.ts(130, 5): 'headers' is declared here.
```

**Comment by ecyrbe** [comment]
Yes this is a breaking change from axios. 
Not zodios fault. revert to axios 1.2.2. cf : https://github.com/axios/axios/pull/5420

**Comment by ecyrbe** [comment]
Please confirm reverting fix your issue.

**Comment by ecyrbe** [comment]
For v11 i find a way to be compatible for this change. so i'll backport this change to v10

**Comment by ecyrbe** [comment]
@vltansky this is fixed in `@zodios/core` `10.7.2`

**Comment by vltansky** [comment]
It's working. Thank you!
